docs/ehrql-tutorial.md

```diff
@@ -2,7 +2,50 @@
 
 ---8<-- 'includes/data-builder-danger-header.md'
 
-This page will contain an [ehrQL](ehrql-intro.md) tutorial. It will
-initially describe how a relatively minimal ehrQL example works.
+In this tutorial, we will walk through a minimal ehrQL example.
+The example will create a dataset of all patients who were born in or before 2000.
+We will walk through the example step-by-step;
+if you want to see the example in full, then you can [skip to the end](#wrapping-up).
```

Comment:
I've used the first-person plural throughout the tutorial ("We..."). I've used the future form in the introduction ("We will walk through") but the present simple in the body ("We import").

Reply:
Important to get @StevenMaude's view on our "voice policy" here, though post-merge would be fine...